### PR TITLE
Mobile: feat show canvas mobile

### PR DESCRIPTION
**Pull request by hoangtranlehuy**
Mobile show channel canvas list and show canvas.
https://github.com/nccasia/mezon/issues/4541

### CODE REVIEW

**Comment by hoangtranlehuy** [comment]
Screenshot 2024-11-14 143112

File: apps/mobile/src/app/navigation/Authentication/stacks/MenuChannelStack.tsx [hunk 1]
```diff
@@ -77,6 +78,17 @@ export function MenuChannelStacks({}: any) {
...
 			<Stack.Screen name={APP_SCREEN.MENU_CHANNEL.ADVANCED_PERMISSION_OVERRIDES} component={AdvancedPermissionOverrides} />
+			<Stack.Screen
+				name={APP_SCREEN.MENU_CHANNEL.CANVAS}
+				component={CanvasScreen}
+				options={{
+					headerTitle: '',
+					headerStyle: {
+						backgroundColor: Colors.bgCharcoal
+					},
+					headerTintColor: Colors.white
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/navigation/Authentication/stacks/MenuChannelStack.tsx]: thêm headerLeftLabelVisible: false cho ios

File: apps/mobile/src/app/components/Canvas/CanvasBoard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import { style } from './styles';
+
+type ScreenChannelCanvas = typeof APP_SCREEN.MENU_CHANNEL.CANVAS;
+export function CanvasScreen({ navigation, route }: MenuChannelScreenProps<ScreenChannelCanvas>) {
+	const { themeValue } = useTheme();
+	const styles = style(themeValue);
+	const { clanId, channelId, canvasId} = route.params;
+	const authState = useSelector(getAuthState);
+
+	const uri = `https://mezon.ai/chat/canvas-mobile/${clanId}/${channelId}/${canvasId}`;
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/components/Canvas/CanvasBoard/index.tsx]: https://mezon.ai lấy từ env

File: apps/mobile/src/app/components/Canvas/CanvasBoard/styles.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { Attributes } from '@mezon/mobile-ui';
+import { StyleSheet } from 'react-native';
+
+export const style = (colors: Attributes) =>
+	StyleSheet.create({
+		Container: {
+			flex: 1
```
Comment (phuocnguyenncc) [inline apps/mobile/src/app/components/Canvas/CanvasBoard/styles.ts]: thêm background Colors.bgCharcoal

**Review by phuocnguyenncc — APPROVED** [review]

**Pull request merged**